server/cpho/jinja2/indicators/view_indicator.jinja2

```diff
@@ -17,17 +25,32 @@
       </thead>
       <tbody>
         {% for period,data_count in data_counts_by_period.items() %}
-          <tr>
-            <th>{{ period }} {{ tdt("records") }}</th>
-            <td>{{ data_count }}</td>
-            <td>
-              <a href="{{ url('view_indicator_for_year', args=[object.id, period.pk]) }}">{{ tdt("See/Edit data") }}</a>
-            </td>
-          </tr>
+          {% if data_count > 0 %}
+            <tr>
+              <th>{{ period }}</th>
+              <td>{{ data_count }}</td>
+              <td>
+                <a href="{{ url('view_indicator_for_period', args=[indicator.id, period.pk]) }}">{{ tdt("See/Edit data") }}</a>
+              </td>
+            </tr>
+          {% endif %}
         {% endfor %}
- 
       </tbody>
     </table>
   </div>
+  <div class="mt-5">
+    <div class="h3">{{ tdt("Add data for new periods") }}</div>
+    <div>
+      <ul>
+        {% for period,data_count in data_counts_by_period.items() %}
+          {% if data_count == 0 %}
+            <li>
+              <a href="{{ url('view_indicator_for_period', args=[indicator.id, period.pk]) }}">{{ tdt("Add data for ") }} {{ period }}</a>
+            </li>
+          {% endif %}
+        {% endfor %}
+      </ul>
+    </div>
+  </div>
```

Comment:
I split up the period selection, 

![Screenshot 2023-07-26 at 17 57 03](https://github.com/PHACDataHub/cpho-phase2/assets/7769215/8f3c6aaa-8920-4b9e-95ce-12de0d942f1b)